README.md

```diff
@@ -370,7 +370,7 @@ $element = $session->element(PHPWebDriver_WebDriverBy::ID, $value);
 
 ## Profiles
 
-*   To send a Firefox profile over the wire to a remote Se server. It is up to you to configure it how you want.
+*   To send a Firefox profile over the wire to a remote The server. It is up to you to configure it how you want.
```

Comment:
I believe the original text was more correct, and not a typo, as "Se" was likely shorthand for "Selenium". Is your fix a typo here (e.g. "The"). Should leave term out making it "remote server" or insert/replace with "Selenium" (or "WebDriver").